config/routes.rb

```diff
@@ -24,4 +24,19 @@
   if Rails.env.development? || Rails.env.dev?
     get "/sandbox", to: "sandbox#index"
   end
+
+  resources :challenges do
+    resource :manage_evaluators, only: [] do
+      collection do
+        get :index
+        post :create
+        delete :destroy
+      end
+    end
+    resources :evaluator_invitations, only: [] do
+      member do
+        post 'resend_invitation'
```

Comment:
nit: this route could be shorter `POST /evaluator_invitations/resend`
```suggestion
        post 'resend'
```